component/org.wso2.carbon.identity.oauth2.dpop/src/main/java/org/wso2/carbon/identity/oauth2/dpop/dao/DPoPJKTDAO.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth2.dpop.dao;
+
+import org.wso2.carbon.identity.oauth2.IdentityOAuth2Exception;
+
+/**
+ * Data Access Object Interface for DPoP JKT
```

Comment:
```suggestion
 * Data Access Object Interface for DPoP JKT.
```
Check other places too.

Reply:
fixed in https://github.com/wso2-extensions/identity-oauth-dpop/pull/2/commits/27c6d0bd0baaef8a186107a5f2eb47a79f4a80f6